README.md

```diff
@@ -53,26 +53,26 @@ You need to set in the [config.ini](config.ini) file the argument "want_soft_lin
 ## Help
 
 <!--readme_update start help -->
-To run yapi with the terminal question installer:
-
- 	yapi console
-
-To pull the newest yapi version from github:
-
- 	yapi update
-
-To get information about yapi:
-
- 	yapi help
-
-To install one of the packages:
-
- 	yapi install <package_to_install>
-
-To recreate the cache:
-
- 	yapi cache
-
+To run yapi with the terminal question installer: 
+ 
+ 	yapi console  
+ 
+To recreate the cache: 
```

Comment:
what does YAPI do with the cache exactly?

Reply:
It creates a scripts.bin file with all infos about packages: description and bash code. If this file doesn't exist it create a new one. I made this to prevent every time call the function:
https://github.com/YetAnotherPackageInstaller/YAPI/blob/0f95546edd38aab6253f60af1b2985777734da7a/modules/cache/cache_manager.py#L44
With small amount of scripts this is insignificant but if the scripts grows this will be necessary.

Reply:
cool, understood!